configure/expose.md

```diff
@@ -0,0 +1,15 @@
+Exposing cluster nodes with dedicated IP addresses
+==============================================================
+
+When Kubernetes creates Pods, each Pod gets its own IP address in the internal virtual network of the cluster. But creating and destroying Pods is a dynamical process, so binding communication between Pods to specific IP addresses would cause problems when something changes over time (as a result of the cluster scaling, maintenance, etc.). 
```

Comment:
```suggestion
When Kubernetes creates Pods, each Pod gets its own IP address in the internal virtual network of the cluster. But creating and destroying Pods is a dynamical process, so binding communication between Pods to specific IP addresses would cause problems when something changes over time (as a result of the cluster scaling, maintenance, etc.). 
Because of this, you should connect to Percona Server for MongoDB via Kubernetes internal DNS names in URI (e.g. `mongodb+srv://userAdmin:userAdmin123456@<cluster-name>-rs0.<namespace>.svc.cluster.local/admin?replicaSet=rs0&ssl=false`). It is strictly recommended.
```